controllers/toolchainstatus/toolchainstatus_controller.go

```diff
@@ -593,8 +593,18 @@ func ExtractStatusMetadata(instance *toolchainv1alpha1.ToolchainStatus) []*Compo
 		cond, found = condition.FindConditionByType(instance.Status.HostOperator.Conditions, toolchainv1alpha1.ConditionReady)
 		if found && cond.Status != corev1.ConditionTrue {
 			result = append(result, &ComponentNotReadyStatus{
-				ComponentType: "",
-				ComponentName: "Host Operator",
+				ComponentType: "Host Operator",
+				ComponentName: "Deployment",
+				Reason:        cond.Reason,
+				Message:       cond.Message,
+			})
+		}
+
+		cond, found = condition.FindConditionByType(instance.Status.HostOperator.RevisionCheck.Conditions, toolchainv1alpha1.ConditionReady)
+		if found && cond.Status != corev1.ConditionTrue {
+			result = append(result, &ComponentNotReadyStatus{
+				ComponentType: "Host Operator",
```

Comment:
same here, `ComponentType` and `ComponentName` looks inverted

Reply:
inverted , please see: https://github.com/codeready-toolchain/host-operator/pull/836/commits/54d573742cad4df097d7652a5ae06a356182acf0